parsers/round-number/README.md

```diff
@@ -0,0 +1,348 @@
+# Round number
+
+## Description
+
+Rounds any measurement with specific precision. 
+You can easily round any measurement value (font size, line height, blur etc.) and specify the number of decimals.
+By specifying a mode, you can control the round function.
+
+## Interface
+
+```ts
+interface parser {
+  "name": "round-number",
+  "options": {
+    keys: Array<string>;
+    precision?: number;
+    mode?: 'down' | 'up' | 'auto';
+  }
+}
+```
+
+### Options
+
+| Parameter              | Required   | Type             | Default    | Description                                                                                                                |
+| ---------------------- | ---------- | --------------   | ---------- | -------------------------------------------------------------------------------------------------------------------------- |
+| `keys`                 | required   | `Array<string>`  |            | List of key path to apply the rounding |
+| `precision`            | optional   | `number`         | `2`        | Number of decimal expected   |
```

Comment:
As this parser is mainly meant to fix pixel values I suggest the default value to be 0.

Reply:
Agreed. @AllanMichay what do you think about that?

Reply:
Yes ok